examples/acr/skaffold.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: skaffold/v1alpha3
+kind: Config
+build:
+  artifacts:
+  - imageName: myregistry.azurecr.io/skaffold-example
+  acr:
+    containerRegistry: myregistry
+    resourceGroup: myresourcegroup
+    credentials:
```

Comment:
Can the credentials be found automatically by Skaffold like it's done for GCB? With GCB, you don't have to configure anything in Skaffold if `gcloud` is properly authenticated to the right project. 

I thing both scenario make sense : 
 + One were you assume the user's machine is properly configured. In that case Skaffold can have the minimal configuration.
 + One were we prefer to configure Skaffold explicitly. We should add that mode to the GCP builder.

Reply:
I could create an Authorizer which uses the bearer token from the azure cli config.

So the priority would be: try to use provided credentials, if they're not present try to authenticate with bearer from the config, if the bearer has expired throw an error. I could try to call the azure cli to refresh the bearer though...